docs/swagger.yaml

```
@@ -494,6 +501,53 @@ paths:
       summary: Updates a license
       tags:
       - license
+  /tempdata/{uuid}:
+    post:
+      description: Create a new temporary data entry with vehicle and user information
+      parameters:
+      - description: UUID of vehicle
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)